pkgs.nix

```diff
@@ -0,0 +1 @@
+import <nixpkgs> {}
```

Comment:
Open says this is the wrong way to fix this problem. All our scripts take a `pkgs` parameter, which can be used to pin everything to a particular nixpkgs. Remove all of the pkgs.nix stuff and resubmit this with only terrapkgs embedded in lib.nix